### ISSUE TITLE
[trusty] postgresql-9.4 postgis-2.1: could not access file "$libdir/postgis-2.2":

### ISSUE THREAD

**Issue by johanneswuerbach** [travis-ci/travis-ci#5509]
When installing postgis for postgres 9.4 manually

``` yaml
sudo: required
dist: trusty
addons:
  postgresql: "9.4"
  apt:
    packages:
      - postgresql-9.4-postgis-2.1
```

https://github.com/travis-ci/travis-ci/issues/5178, I receive now:

```
CREATE EXTENSION postgis
error: could not access file "$libdir/postgis-2.2": No such file or directory
```

Manually installing `postgis-2.2` resolved the issue:

``` yaml
before_install:
 - export DEBIAN_FRONTEND=noninteractive;
   sudo -E apt-get -yq update &>> ~/apt-get-update.log;
   sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install postgresql-9.4-postgis-2.2
```

**Comment by carhartl** [comment]
Is that because the postgresql-9.4-postgis-2.1 package nevertheless downloads postgresql-9.4-postgis-scripts along with it, and that, in its latest version assumes postgis 2.2? 

https://packages.debian.org/jessie-backports/all/postgresql-9.4-postgis-scripts/filelist

**Comment by denniswambua** [comment]
+1

**Comment by arthurlogilab** [comment]
Another fix is to downgrade to ```postgresql-9.4-postgis-scripts=2.1.4+dfsg-3``` sticking to postgis-2.1
